GCRCatalogs/alphaq.py

```diff
@@ -70,14 +76,23 @@ def _subclass_init(self, filename, **kwargs):
             'velocity_z':         'vz',
         }
 
+        if(catalog_version < StrictVersion('2.1.1')):
+            self._quantity_modifiers.update({
+                'disk_sersic_index':  'diskSersicIndex',
+                'bulge_sersic_index': 'spheroidSersicIndex',
+            })
+            del self._quantity_modifiers['ellipticity_1']
+            del self._quantity_modifiers['ellipticity_2']
+
         if catalog_version == '2.0': # to be backward compatible
```

Comment:
Use `StrictVersion` for comparison
`if catalog_version <= StrictVersion('2.0'):`

Reply:
Okay, it's in there.